classes/opponentPhaseCharge.lua

```diff
@@ -19,6 +19,7 @@ function opponentPhaseCharge.new()
 	self.chargeTime = 1
 	self.chargeUpTime = 1
 	self.CHARGE_SPEED = 500
+	self.angle = math.atan2(state.player.y - self.y, state.player.x - self.x)
```

Comment:
I added this line for a few phases

On creation, auto face towards the player. This prevents any graphical glitch where for one frame, the opponent is facing downwards, and on the next, facing the player